service/frontend/handler.go

```diff
@@ -1150,9 +1150,9 @@ func getSerializedGetHistoryToken(persistenceToken []byte, runID string, history
 	events := history.GetEvents()
 	if len(persistenceToken) > 0 && len(events) > 0 && events[len(events)-1].GetEventId() < nextEventID-1 {
 		token := &getHistoryContinuationToken{
-			runID:            runID,
-			nextEventID:      nextEventID,
-			persistenceToken: persistenceToken,
+			RunID:            runID,
```

Comment:
can you also add a unit test for this helper that makes sure the serialized json is in the expected format ?